src/pages/ExamsPage/components/AttemptList.test.jsx

```diff
@@ -2,10 +2,22 @@ import { render, screen } from '@testing-library/react';
 
 import AttemptList from './AttemptList';
 
+import * as hooks from '../hooks';
+
 // nomally mocked for unit tests but required for rendering/snapshots
 jest.unmock('react');
 
+jest.mock('../hooks', () => ({
+  useDeleteExamAttempt: jest.fn(),
+}));
+
+const mockMakeNetworkRequest = jest.fn();
+
 describe('AttemptList', () => {
+  beforeEach(() => {
```

Comment:
NIT: I don't think restore is doing anything since we aren't altering mocks in any of these test cases. Also `mockMakeNetworkRequest` is slightly inaccurate since the hook returns an anonymous function that executes makeNetworkRequest. I'd recommend just mocking this with a jest.fn() or calling it `mockDeleteExamAttempt` to avoid confusion.